packages/js/src/classic-editor/replacement-variables/configurations.js

```diff
@@ -1,29 +1,33 @@
 import { __, sprintf } from "@wordpress/i18n";
-import { replacementVariableConfigurations } from "@yoast/seo-integration";
+import { replacementVariableConfigurations, SEO_STORE_NAME } from "@yoast/seo-integration";
 import { get, map } from "lodash";
+import { select } from "@wordpress/data";
 
 /**
  * Gets the parent title from the select element.
  *
- * @param {HTMLElement} select The select input.
+ * @param {HTMLElement} selectElement The select input.
  *
  * @returns {string} The parent title.
  */
-const getParentTitle = ( select ) => {
-	const selectedValue = get( select, "value", "" );
+const getParentTitle = ( selectElement ) => {
```

Comment:
Note for reviewers: I changed this argument name from `select` to `selectElement` since it could otherwise interfere with the `select` import I added in this PR.